tests/common/mellanox_data.py

```diff
@@ -26,6 +26,32 @@
         },
         "ports": {
             "number": 32
+        },
+        "thermals": {
+            "cpu_core": {
```

Comment:
The cpu_core is repeated here. Can we merge the "start":0 key value pair to the above cpu_cores key?

Reply:
Here "cpu_core" is not about how many cpu cores on this switch, but how many thermal zones on  cpu cores.